lib/install.js

```diff
@@ -47,7 +47,7 @@ function* install(parentDir, pkg, options) {
 function* _install(parentDir, pkg, options) {
   // default install latest version
   if (!pkg.version) {
-    pkg.version = '*';
+    pkg.version = 'latest';
```

Comment:
![image](https://cloud.githubusercontent.com/assets/985607/15468527/5066d47c-2117-11e6-8e21-5378c407aad0.png)

we can't fetch scope package with tag from npm's registry. WTF.